src/Equinor.ProCoSys.IPO.WebApi/Synchronization/BusReceiverService.cs

```diff
@@ -31,19 +35,39 @@ public async Task ProcessMessageAsync(PcsTopic pcsTopic, Message message, Cancel
             {
                 case PcsTopic.Project:
                     var projectEvent = JsonSerializer.Deserialize<ProjectTopic>(messageJson);
-
+                    _telemetryClient.TrackEvent("IPO Bus Receiver",

```

Comment:
Consider using constant strings for all static telemetry strings to avoid differences.